new-lamassu-admin/src/pages/Services/Services.js

```diff
@@ -101,40 +126,113 @@ const Services = () => {
   const getValidationSchema = ({ code, getValidationSchema }) =>
     getValidationSchema(accounts[code])
 
+  const isServiceEnabled = service =>
+    !R.isNil(accounts[service.code]) && Boolean(accounts[service.code]?.enabled)
+
+  const isServiceDisabled = service =>
+    !R.isNil(accounts[service.code]) && !accounts[service.code]?.enabled
+
+  const enabledServices = R.filter(isServiceEnabled, R.values(schemas))
+  const disabledServices = R.filter(isServiceDisabled, R.values(schemas))
+  const usedServices = R.filter(
+    it => isServiceDisabled(it) || isServiceEnabled(it),
+    R.values(schemas)
+  )
+  const unusedServices = R.filter(
+    it => !isServiceDisabled(it) && !isServiceEnabled(it),
+    R.values(schemas)
+  )
```

Comment:
The `schemas` aren't many, but if you think it's worth it, it's possible to optimize this bit of code into two `R.partition()`s. Something like this:

```js
const [enabledServices, limbo] = R.partition(isServiceEnabled, R.values(schemas))
const [disabledServices, unusedServices] = R.partition(isServiceDisabled, limbo)
const usedServices = R.concat(enabledServices, disabledServices)
```